File: Main Project/Backend .net/RentIT/Models/Role.cs
using System;
using System.Collections.Generic;

namespace RentIT.Models
{
    public partial class Role
    {
        public Role()
        {
            Logins = new HashSet<Login>();
        }

        public int RoleId { get; set; }
        public string? Name { get; set; }

        public virtual ICollection<Login> Logins { get; set; }
    }
}

File: Karan/BackUpBackEnd/HomeAppliancesSpring/src/main/java/com/example/demo/entities/Role.java
package com.example.demo.entities;

import java.util.HashSet;
import java.util.Set;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import jakarta.persistence.CascadeType;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.OneToMany;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "role")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Role {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int role_id;
	
	
	@Column
	String name;
	
	@JsonIgnoreProperties("role")
	@OneToMany(mappedBy = "role",cascade = CascadeType.ALL)
	Set<Login> logins= new HashSet<Login>();
}
